requirements_for_test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 flake8==3.8.4
+flake8-bugbear==20.11.1
```

Comment:
Do we want to use flake8-print too?

Reply:
We use `print` in a few different files in this repo, for example [in `commands.py`](https://github.com/alphagov/notifications-api/blob/7152ac7cba4a8f1cf9c9b378d3fe6bae0cf75b4f/app/commands.py) so I decided against it.